libs/ui/elements/src/lib/internal-link-card/internal-link-card.component.stories.ts

```diff
@@ -161,7 +168,7 @@ export const WithoutGeodataIndicator: StoryObj<InternalLinkCardComponentWithFavo
   {
     args: {
       ...Primary.args,
-      isGeodata: false,
+      // isGeodata: false, // TODO: update storybook
```

Comment:
Do you still want to address this in this PR?

Reply:
yes, forgot to update storybook after refactor, thx